app/views/projects/_project-card.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= link_to project, class: 'relative block shadow-lg transform transition-transform duration-100 hover:scale-105' do %>
   <div class="w-full bg-center bg-cover" style="height: 130px; background-image: url(<%= project.cover_photo(local_assigns[:group]) %>);"></div>
 
-  <% if project.accepting_volunteers? %>
+  <% if project.accepting_ss? %>
```

Comment:
my command key isn't working that well either

Reply:
oh wow.. lol. i keep on switching between Mac and Windows and forget which key i'm using.

Reply:
It's kind of strange the tests passed on this.. going to investigate..